net.py

```diff
@@ -1,172 +0,0 @@
-import numpy as np
-
-class Net:
-    def __init__(self, layers):
-        self.W = []
-        self.dW = []
-        self.b = []
-        self.db = []
-        self.A_saved = []
-        self.Z_saved = []
-        self.dA = []
-        self.dZ = []
-        self.backup = []
-        self.L = len(layers) - 1
-        self.layers = layers
-        # -1 are appended to arrays in order to assign n-th index to n-th layer, e.g. W2 = W[2]
-        self.W.append(-1)
-        self.b.append(-1)
-        self.A_saved.append(-1)
-        self.Z_saved.append(-1)
-        self.dA.append(-1)
-        self.dW.append(-1)
-        self.db.append(-1)
-        self.dZ.append(-1)
-        for i in range(1,self.L+1):
-            #print(np.sqrt(1/layers[i-1]))
-            self.W.append(np.random.randn(layers[i], layers[i-1]) * 0.01) #np.sqrt(2/layers[i-1]))
-            self.b.append(np.zeros((layers[i], 1)))
-            self.A_saved.append(-1)
-            self.Z_saved.append(-1)
-            self.dW.append(-1)
```

Comment:
You don't have to initialize empty field and then append. Maybe just create one element list (it's not C, it isn't cost so much)